excel/main.go

```diff
@@ -33,6 +35,17 @@ func main() {
 		err = commands.AddWorksheetRow(context.Background(), os.Getenv("WORKBOOK_ID"), os.Getenv("WORKSHEET_ID"), os.Getenv("CONTENTS"))
 	case "createWorksheet":
 		err = commands.CreateWorksheet(context.Background(), os.Getenv("WORKBOOK_ID"), os.Getenv("NAME"))
+	case "getDate":
+		serialStrings := strings.Split(os.Getenv("SERIALS"), "|")
+		serials := make([]int, len(serialStrings))
+		for i, v := range serialStrings {
+			val, err := strconv.Atoi(strings.TrimSpace(v))
+			if err != nil {
+				break
```

Comment:
I feel like we should print an error here, and then either continue with the loop or just exit(1).

Reply:
Hm.. yeah i dont think I thought this through all the way because it doesn't break all the way out of the switch. I've just pushed a change that should address this.